README.md

```diff
@@ -27,6 +28,10 @@ NOTE: If you are unable to purchase the Armchair Analysis data, [Ben Dilday](htt
 - pandas
 - scikit-learn
 
+## Node.js package requirements
```

Comment:
is this useful given that these are already enumerated in `package.json` (and will be installed on `npm install`)?